HelpSource/Classes/IdentityDictionary.schelp

```diff
@@ -22,6 +22,52 @@ The contents of a Dictionary are strong::unordered::. You must not depend on the
 
 IdentityDictionary is often used to assign names to instances of a particular class. For example, the proxy classes ( link::Classes/Pdef::, link::Classes/Pdefn::, link::Classes/Tdef::, link::Classes/Ndef:: ), and link::Classes/NodeWatcher:: all have class variables named all implemented as IdentityDictionaries.
 
+Subsection:: 'parent' and 'proto' variables
+
+IdentityDictionary has three levels of content: the dictionary itself, a code::proto::, and a code::parent::. code::proto:: and code::parent:: exist primarily for IdentityDictionary's subclass link::Classes/Event::, which uses the code::parent:: to store action functions that will be used when an event is played. Users may put additional default values into the code::proto::.
+
+Looking up a key within a dictionary first checks the dictionary itself. If the key is not found at this level, it looks in the code::proto::, and if still not found, it looks in the parent.
+
+These extra levels are meant for common, default values that should be the same across many dictionary instances.
+
+list::
+## code::proto:: and code::parent:: values are not shown when posting the dictionary.
+## Copying an IdentityDictionary transfers a reference to the parent and proto into the copy. It is a reference, not a copy: changing the copy's parent also affects the parent of the original. This is for efficiency.
```

Comment:
(suggestion:)

this is more efficient than a deep copy, and also allows to inherit the dynamic relation to the parent.